internal/task/task/common/create_container.go

```diff
@@ -279,6 +280,7 @@ func NewCreateContainerTask(curveadm *cli.CurveAdm, dc *topology.DeployConfig) (
 		Init:        true,
 		Name:        hostname,
 		Privileged:  true,
+		Remove:      !debugmode,
```

Comment:
here is a problem, the service container exit but don't delete it forever util delete it manually.